src/containers/Proposal/Vetted/helpers.js

```diff
@@ -7,58 +7,29 @@ import {
 } from "src/constants";
 
 export const tabValues = {
-  IN_DISCUSSION: "In Discussion",
-  VOTING: "Voting",
+  UNDER_REVIEW: "Under Review",
   APPROVED: "Approved",
   REJECTED: "Rejected",
   INELIGIBLE: "Abandoned"
 };
 
 export const noProposalMessage = "No proposals available";
 
-/**
- * Returns an array of proposal tokens for the given tab option
- * @param {string} tabOption
- * @param {Object} proposalsTokens
- * @returns {Array} array of proposals tokens
- */
-export const getProposalTokensByTabOption = (tabOption, proposalsTokens) => {
-  if (!proposalsTokens) return [];
-  const { pre, started, approved, rejected, ineligible } = proposalsTokens;
-  switch (tabOption) {
-    case tabValues.IN_DISCUSSION:
-      return pre;
-    case tabValues.VOTING:
-      return started;
-    case tabValues.APPROVED:
-      return approved;
-    case tabValues.REJECTED:
-      return rejected;
-    case tabValues.INELIGIBLE:
-      return ineligible;
-    default:
-      return [];
-  }
-};
-
-/**
- * Returns an onject mapping the proposals tokens to each tab label
- * @param {array} tabLabels
- * @param {object} proposalsTokens
- */
-export const mapProposalsTokensByTab = (tabLabels, proposalsTokens) => {
-  return tabLabels.reduce((map, tab) => {
-    return {
-      ...map,
-      [tab]: getProposalTokensByTabOption(tab, proposalsTokens)
-    };
-  }, {});
+export const statusByTab = {
+  [tabValues.UNDER_REVIEW]: [
+    PROPOSAL_VOTING_ACTIVE,
+    PROPOSAL_VOTING_NOT_AUTHORIZED
+  ],
```

Comment:
```suggestion
  [tabValues.UNDER_REVIEW]: [
    PROPOSAL_VOTING_ACTIVE,
    PROPOSAL_VOTING_AUTHORIZED,
    PROPOSAL_VOTING_NOT_AUTHORIZED
  ],
```
Once you fix the inventory action on [src/actions/api.js#L590-593](https://github.com/decred/politeiagui/blob/595f99ac95d555d09973407333b2c4c44393c4e8/src/actions/api.js#L590-L593), you can add the `authorized` status here